test/e2e/harness.ts

```diff
@@ -0,0 +1,34 @@
+import { Page } from "@recordreplay/playwright";
+
+declare global {
+  const app: any;
+}
+
+export default class TestHarness {
+  constructor(private readonly page: Page) {}
+
+  async start() {
+    await this.page.waitForFunction(() => window.app?.actions);
+    this.page.evaluate(() => window.app.actions.setViewMode("dev"));
+    await this.page.waitForFunction(() => {
+      const loadedRegions = app.selectors.getLoadedRegions();
+      const fullyLoaded =
+        loadedRegions &&
+        loadedRegions.loading.length > 0 &&
+        loadedRegions.loaded.length > 0 &&
+        loadedRegions.loading[0].end.point === loadedRegions.loaded[0].end.point;
```

Comment:
We don't need to wait for indexing as well here?

Reply:
Apparently not, I copied this verbatim from the old harness: https://github.com/RecordReplay/devtools/blob/09040ae4b06002de3ff629964ac120b2289caeee/src/test/harness.js#L86-L89